### PR TITLE
feat(project): update project structure

### PR DESCRIPTION
**Pull request by saitofun**

### CODE REVIEW

File: project/project.go [hunk 1]
```diff
@@ -16,8 +16,72 @@ import (
...
+func (c *Config) DefaultConfigData() *ConfigData {
+	return c.defaultData
+}
+
+func (c *Config) GetConfigDataByVersion(version string) *ConfigData {
+	return c.versions[version]
+}
+
+type ConfigData struct {
+	DatasourceURI   string            `json:"datasourceURI"`
```
Comment (huangzhiran) [inline project/project.go:84]: DatasourceURI should in Config field, for a project should only have one data source at a time

File: project/project.go [hunk 2]
```diff
@@ -16,8 +16,72 @@ import (
...
 	VMType          vm.Type           `json:"vmType"`
 	Output          output.Config     `json:"output"`
 	Aggregation     AggregationConfig `json:"aggregation"`
```
Comment (huangzhiran) [inline project/project.go:87]: del Aggregation, for not used currently

File: project/project.go (hunk 1 above)
Comment (saitofun) [reply project/project.go:84]: `ConfigData` will inherit from `Config` if this field is not assigned.

File: project/project.go (hunk 2 above)
Comment (saitofun) [reply project/project.go:87]: keep it now?

**Review by huangzhiran — APPROVED** [review]

**Pull request merged**